setup.py

```diff
@@ -570,17 +569,16 @@ def _embed_libomp(self):
                 continue
             target_lib = os.path.join(self.build_lib, "torch", "lib", omp_lib_name)
```

Comment:
Does this change also affects arm64 builds ?

Reply:
No, It's written specifically to target only Darwin. On Linux, two libgomp.dylib can interop with each other nicely